CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 2.4.0
```

Comment:
Feels like this should perhaps be a major version?

Reply:
Yeah, maybe - essentially it's a bug fix though

Reply:
But I guess updating the major version is safest of all three